docs/source/model_design/vivarium_overview/data_sources/index.rst

```diff
@@ -134,29 +134,99 @@ For reference, `the decomposition rules for GBD 2021 can be found here <https://
-----------------------------------------------
+----------------------------------
 
-.. todo::
+Generally, GBD shared functions offers greater flexibility in querying GBD data than Vivarium Inputs, but requires specification of detailed IDs that are not human-readable and requires translation with get_ids. Vivarium Inputs offers less flexibility in favor of the convenience of returning a human-readable version of the most relevant data for running Vivarium simulations and compatibility with required Vivarium Artifact formatting. Therefore, GBD shared functions may be the code base of choice to use when taking deep dives into GBD data, and Vivarium Inputs when preparing GBD data for Vivarium simulations. Some specific comparisons are summarized in the table below for additional specific considerations of the differences betweent the two code bases.
 
-   Discuss how:
-
-   - Vivarium inputs converts IDs to names, which may be convenient for pairing  formatting with vivarium outputs
-   - Vivarium inputs may have some wrappers/transformations in get_measures versus get_raw_data
-   - Vivarium inputs is available for finalized GBD rounds only (can only pull intermediate results using shared functions)
+.. list-table::
+   :header-rows: 1
 
-   Maybe link to functions to help us convert between shared functions and vivarium formatting so that translating between the two isn't so annoying 
\ No newline at end of file
+   *  - Topic
+      - GBD Shared Functions
+      - Vivarium Inputs
+   *  - GBD round
+      - Able to specify any GBD round/release; useful for noting and comparing major changes between rounds
+      - Returns most recent complete GBD round/release only
+   *  - DALYs
+      - Returns YLD, YLL, DALY estimates
+      - Does not return YLD, YLL, or DALY estimates
+   *  - Metrics
+      - Returns counts, rates, and prevalence estimates
+      - Returns rate estimates with the exception of population structure, which are in counts; convenient
+   *  - Summary values
+      - Can return mean, upper, and lower estimates using get_outputs
+      - Returns draw-level estimates only
+   *  - Age/sex/location specificity
+      - Allows for specification across all these parameters, allows for grouping (via get_outputs) and/or aggregation (via make_custom_aggregates) across demographic categories
+      - Returns *all* most-detailed age and sex estimates. Supports only one location at a time.
+   *  - Format
+      - Generally uses ID numbers that are not human-readable before pairing with get_ids information
+      - Converts to human readable entity names rather than IDs and is compatible with formatting required for vivarium Artifacts and simulations
+
+.. note::
+
+   To convert between GBD shared function entity names (such as cause_name) to the entity name in Vivarium inputs, convert the GBD shared function entity name to all lower case and replace spaces with underscores. Python code do this is shown below:
```

Comment:
How confident are we that this is always true?

Reply:
I think this is currently always true. We could expose a function to be used in the GBD Mapping generator when creating the entities in the first place that could be referenced here if we are concerned about this.

Reply:
Is this the appropriate line as you see it? https://github.com/ihmeuw/gbd_mapping/blob/1511a4181d45ece5c146b6c5934e5d8b35774409/src/gbd_mapping_generator/util.py#L167

Reply:
Yes I believe so.